catalogue/templates/catalogue/modals/compare/compare_button.html

```diff
@@ -0,0 +1,86 @@
+{% load list_fields %}
+
+<div class="compare-button" style="display: flex; flex-direction:row;">
+  {% if products.count %}
+  <div class="dropdown is-up" dropdown-menu-min-width="20rem" style="flex: 1 1 auto;">
+    <div class="dropdown-trigger">
+      <button class="button is-success" aria-haspopup="true" aria-controls="dropdown-menu7">
+        <span class="bellnumbers">
+          {{ products.count }}
+        </span>
+        <span class="icon">
+          <i class="fas fa-chart-pie"></i>
+        </span>
+        {% if products.count < 4 %}
+        <span>Vergleich anzeigen</span>
+        {% else %}
+        <span>Warenkorb voll</span>
+        {% endif %}
+      </button>
+    </div>
+    <div class="dropdown-menu" id="dropdown-menu7" role="menu">
+      <div class="dropdown-item">
+        <nav class="panel is-success">
```

Comment:
maybe remove the panel? It's a bit too much and `dropdown-content` is missing, which leads to additional padding.

Here is a snippet with panel replaced by dropdown-item:

```html
    <div class="dropdown is-up">
      <div class="dropdown-trigger">
        <button class="button is-success" aria-haspopup="true" aria-controls="dropdown-menu7">
        <span class="bellnumbers">
          {{ products.count }}
        </span>
          <span class="icon">
          <i class="fas fa-chart-pie"></i>
        </span>
          {% if products.count < 4 %}
            <span>Vergleich anzeigen</span>
          {% else %}
            <span>Warenkorb voll</span>
          {% endif %}
        </button>
      </div>
      <div class="dropdown-menu" id="dropdown-menu7" role="menu">
        <div class="dropdown-content">
          {% for p in products %}
            <div class="dropdown-item">
              <button
                  class="delete panel-icon"
                  hx-delete="{% url 'catalogue:delete_cart' p.id %}?c={{ c_pk }}"
                  hx-target="#shopping_card"
                  hx-swap="innerHTML"
              >
              </button>
              <a href="{% url 'catalogue:detail' p.id %}">{{ p.name }}</a>
            </div>
            <div class="dropdown-divider"></div>
          {% endfor %}
          </a>
          {% if products.count > 1 %}
            <div class="dropdown-item">
              <a class="button is-fullwidth is-success is-small"
                 href="{% url 'catalogue:compare' %}?{% for v in products %}id={{ v.id }}&{% endfor %}">
                <span>Zum Vergleich</span>
                <span class="icon">
                  <i class="fas fa-arrow-right"></i>
                </span>
              </a>
            </div>
          {% endif %}
        </div>
      </div>
    </div>
```